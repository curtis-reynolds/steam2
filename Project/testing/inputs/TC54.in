login
buystdwcredit
buy
existinggame2
buystdwcredit
