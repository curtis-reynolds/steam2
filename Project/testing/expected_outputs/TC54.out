Login successful.
Error: You are the seller of the game.
